apps/ex_nvr_web/lib/ex_nvr_web/live/recordings_list_live.ex

```diff
@@ -194,6 +208,10 @@ defmodule ExNVRWeb.RecordingListLive do
      |> push_patch(to: Routes.recording_list_path(socket, :list, params))}
   end
 
+  defp subscribe_to_recording_events() do
+    PubSub.subscribe(ExNVR.PubSub, @recordings_topic)
```

Comment:
I think it's better to move this function to `recordings` contexts for two reason:

- You hide the name of the topic
- It's easier for other processes to subscribe to this topic instead of duplicating this piece of code